extra/1-radio-stations.js

```diff
@@ -26,42 +36,58 @@
  */
 // `getStations` goes here
 
+function getStations() {
+
+  
+  let station = [];
+
+	let frequency= getAllFrequencies();
```

Comment:
This shouldn't be indented - it should be the same indentation as `let station = [];`